### PR TITLE
Blockbase: Make the top margins consistent

### LINKED ISSUE #4722: Blockbase: Under-header margin is incorrect
The spacing between a the site header and the page/post title (when there is one title) should be 80px. When you click through from the first post in a blog feed to the post itself, the title should remain in the same place. 

Blog/Archive:

Current|Intended
---|---
Screen Shot 2021-09-28 at 09 44 04|Screen Shot 2021-09-28 at 09 42 35

Single Post: 

Current|Intended
---|---
Screen Shot 2021-09-28 at 09 43 44|Screen Shot 2021-09-28 at 09 43 13

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This adds some top margin to the content of Blockbase, and also makes the top margin consistent between single and index templates:

Screenshot 2021-09-28 at 17 37 19

Screenshot 2021-09-28 at 17 37 13

I have taken a fix from Quadrat and applied it to all Blockbase themes as it felt generic. I've also opened a PR in Gutenberg to address this.

#### Related issue(s):
Fixes https://github.com/Automattic/themes/issues/4722

### CODE REVIEW

**Review by kjellr — APPROVED** [review]
👍 Thanks!

File: blockbase/sass/ponyfill.scss [hunk 1]
```diff
@@ -29,3 +29,6 @@
 @import "blocks/video";
 @import "blocks/columns";
 @import "post/meta";
+.wp-block-post-template {
```
Comment (jffng) [inline blockbase/sass/ponyfill.scss]: Did you mean to import the `blocks/post-template` partial?

Comment (scruffian) [reply blockbase/sass/ponyfill.scss]: oh yeah duh 🤦

Comment (scruffian) [reply blockbase/sass/ponyfill.scss]: done in aa4e21f7e535497425abb6bb4bae192447a65973

**Review by jffng — APPROVED** [review]

**Pull request merged**